app/models/annotation.rb

```diff
@@ -20,6 +20,11 @@ class Annotation < ActiveRecord::Base
   has_many :annotation_tag_maps, dependent: :delete_all
   has_many :annotation_tags, through: :annotation_tag_maps
 
+  has_many :annotation_lists, through: :list_annotations_map
+  has_many :list_annotations_map, foreign_key: :annotation_id, primary_key: :annotation_id, dependent: :destroy
+  has_many :canvases, through: :annotation_lists
+  has_many :annotation_layers, through: :annotation_lists
```

Comment:
Since you have annotation_layers method defined below, should we remove this association?

Reply:
Yes, this is redundant. I'll delete it!